internal/compiler/llr/lower_to_item_tree.rs

```diff
@@ -655,6 +640,8 @@ fn get_property_analysis(elem: &ElementRc, p: &str) -> crate::object_tree::Prope
 
 fn lower_repeated_component(
     elem: &ElementRc,
+    parent: Option<ElementRc>,
+    sub_component: &SubComponent,
```

Comment:
This is unclear that this relate only to component container.
This should be properly documented and otherwise properly named

```suggestion
    parent_component_container: Option<ElementRc>,
    sub_component: &SubComponent,
```

But I'm wondering if it is possible to pass the container_item_index instead

Reply:
I like that, will be in the next push.